### PR TITLE
Add minor updates

### PR DESCRIPTION
**Pull request by jtapia**
- Update ruby syntaxis
- Update configuration files
- Add `es-MX` translation file
- Use `Spree` translation prefix
- Fix `edit` page on admin

### CODE REVIEW

**Comment by fredericboivin** [comment]
Thanks for the contribution,

My first suggestion would be to add a more descriptive commit message & split the changes into smaller commits if applicable

Will review the individual changes as soon as possible

**Comment by jtapia** [comment]
@dangerdogz what do you think?

**Comment by fredericboivin** [comment]
The 2.X releases include several breaking changes that we need to look into and the test suite is unstable on master. Can you extract the ActiveShipping changes as a separate PR? The remaining changes are much more straightforward and I can look into the build failures in the meantime

**Comment by jtapia** [comment]
@dangerdogz done

File: Gemfile [hunk 1]
```diff
@@ -14,7 +14,8 @@ gem 'pg', '~> 0.21'
 gem 'mysql2'
 
 group :development, :test do
-  gem "pry-rails"
+  gem 'byebug'
```
Comment (fredericboivin) [inline Gemfile]: Why is this dependency needed? This can be easily added in the host application if needed

**Review by fredericboivin — CHANGES_REQUESTED** [review]
The changes to `active_shipping_settings/edit` breaks the extension for every version of Solidus prior to 2.4.0 as this is when the `preference_fields` partial were introduced

**Comment by jtapia** [comment]
@dangerdogz removed

**Comment by jtapia** [comment]
@dangerdogz for the `active_shipping_settings/edit` issue, I propose this https://github.com/solidusio-contrib/solidus_active_shipping/pull/49/commits/beffaace8cf4ed198242659cd867f8b5fbad8e06, what do you think?

**Comment by fredericboivin** [comment]
Considering the status of this extension, I don't think the value added by the new partials warrants the maintenance burden of having to support different views for different versions.

I would be happy to merge the other commits otherwise 👍

**Comment by jtapia** [comment]
@dangerdogz I removed the latest commit with the view changes, let me know if you think that I need to do anything else here, thanks

**Pull request merged**